easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,23 @@
+name = 'udocker'
+version = '1.1.1'
+versionsuffix = '-Python-2.7.14'
+
+easyblock = "PythonPackage"
+
+homepage = 'https://github.com/indigo-dc/udocker'
+description = """A basic user tool to execute simple docker containers in batch
+		 or interactive systems without root privileges"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+##toolchainopts = {'pic': True}
+
+source_urls = [('https://github.com/indigo-dc/udocker/archive/v%(version)s.tar.gz', 'download')]
+sources = [SOURCE_TAR_GZ]
+
+use_pip = True
+
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
```

Comment:
just to report that this easyconfig is about ready to work. module its the sanitycheck, via `eb --from-pr 5770 -r`; 

I only had to resort to this trick to let it pass (I can feel the hot breath of @boegel on my neck over this hack :) 
* `options = {'modulename': 'string'}`

then we are able to obtain (@A1ve5 : could you implement that `--version`? it's a nice to have):
```
[fgeorgatos@somewhere ~]$ ml !$;!!
ml udocker;which udocker
~/.local/easybuild/software/udocker/1.1.1-foss-2017b-Python-2.7.14/bin/udocker
[fgeorgatos@somewhere ~]$ udocker --version
Error: parsing command line, use: udocker help
```

seriously, this requires minimal wrap-up effort to get it into next release... thanks @A1ve5

Reply:
thanks for this @fgeorgatos 
On udocker we try to mimic docker command syntax, so "udocker version" should work.
Nevertheless I'll try to cover the "--version" option as well.

Reply:
On devel branch now: https://github.com/indigo-dc/udocker/commit/083b727aec536fee5b49201862aefa6a54dc7ab7

Reply:
@fgeorgatos We're not savages, you can bypass the `import` check using this:

```python
options = {'modulename': False}
```

;-)